packaging/tests/tas-cmis/pom.xml

```diff
@@ -1,27 +1,28 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.alfresco.tas</groupId>
-    <artifactId>alfresco-community-repo-cmis-test</artifactId>
-    <name>cmis test</name>
-    <packaging>jar</packaging>
-
+    <artifactId>cmis</artifactId>
```

Comment:
Currently I've used the existing group id and artifact id from the existing library project, but have updated the version to match community repo (i.e. alfresco-tas-cmis 1.34 will be followed by alfresco-tas-cmis 17.129). I felt this was the right decision because it will mean we don't need to change other places that use the library - Dependabot should automatically update them.